### PR TITLE
Update SignalR Client Features matrix

### PR DESCRIPTION
**Pull request by anurse**
A few updates:

* Added transports
* Added Azure SignalR support - every client supports it, but we do get questions like "can I use the service with Java?" so it's helpful to confirm that in the matrix, and technically the client has to do some (very minimal) work to support it. I put this at the top, because it's so awesome 😉, and that messed up the visual diff a bit (sorry!).
* Added note about issue tracking automatic reconnect

### CODE REVIEW

File: aspnetcore/signalr/client-features.md [hunk 1]
```diff
@@ -16,9 +16,15 @@ The table below shows the features and support for the clients that offer real-t
 
 | Feature | .NET Core | JavaScript | Java |
 | ---- | :-: | :-: | :-: |
+| Azure SignalR Service Support |✔|✔|✔|
 | [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔|✔|
 | [Client-to-server Streaming](xref:signalr/streaming)          |✔|✔|✔|
 | Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |✔|✔| |
+| WebSockets Transport |✔|✔|✔|
+| Server-Sent Events Transport |✔|✔| |
```
Comment (anurse) [inline aspnetcore/signalr/client-features.md]: FYI, we have no plans to support SSE in Java (and are exploring ways to phase it out of all clients), so I didn't reference an issue for it in the list below.

File: aspnetcore/signalr/client-features.md [hunk 2]
```diff
@@ -16,9 +16,15 @@ The table below shows the features and support for the clients that offer real-t
...
+| Long Polling Transport |✔|✔|✔|
+
+* Support for automatic reconnect in the Java client is tracked in [our issue tracker](https://github.com/aspnet/AspNetCore/issues/8711).
```
Comment (anurse) [inline aspnetcore/signalr/client-features.md]:
```suggestion
Support for automatic reconnect in the Java client is tracked in [our issue tracker](https://github.com/aspnet/AspNetCore/issues/8711).
```

File: aspnetcore/signalr/client-features.md [hunk 3]
```diff
@@ -14,11 +14,19 @@ uid: signalr/client-features
 
 The table below shows the features and support for the clients that offer real-time support.
 
-| Feature | .NET Core | JavaScript | Java |
+| Feature | .NET | JavaScript | Java |
```
Comment (anurse) [inline aspnetcore/signalr/client-features.md]: Also, the client is for .NET Standard and supports .NET Framework, Xamarin, .NET Core, Unity, etc. So calling it ".NET Core" isn't really appropriate. Changed to `.NET`

**Comment by davidfowl** [comment]
Don't we need versions in here?

**Comment by halter73** [comment]
The document has `monikerRange: '>= aspnetcore-3.0'` in its metadata.

**Comment by davidfowl** [comment]
That's not useful for client versions. For example, when we introduce streaming in the java client, or reconnect in the java client. ASP.NET Core 3.0 is insufficient.

**Comment by anurse** [comment]
Perhaps if the matrix includes a minimum version number instead of just a check box? I can sketch that out.

**Comment by anurse** [comment]
https://github.com/aspnet/AspNetCore.Docs/pull/14744

**Pull request merged**